src/rebar3_hex_publish.erl

```diff
@@ -387,6 +387,8 @@ format_build_tools(BuildTools) ->
 update_versions(ConfigDeps, Deps) ->
     [begin
          case lists:keyfind(binary_to_atom(N, utf8), 1, ConfigDeps) of
+             {_, V} when is_binary(V); is_list(V) ->
```

Comment:
this clause obscures the one after since you kept the is_list(V) guard, and won't be converting the value of a list to a binary anymore. I'm not sure this is what you intended?

Reply:
It wasn't, just a mistake when pushing. I'll fix it.